resources/compass/values.yaml

```diff
@@ -4,20 +4,20 @@ global:
       path: eu.gcr.io/kyma-project/incubator
     connector:
       dir:
-      version: "1.3.0-rc1"
+      version: "1.3.0-rc2"
```

Comment:
Please create PR to compass to its release branch, because these images won't exist

Reply:
https://github.com/kyma-incubator/compass/pull/133